challenge.rb

```diff
@@ -57,7 +65,22 @@ def ac_monitor current_temp, ac_working, desired_temp
 # - return "fizzbuzz" if `num` is evenly divisible by both 3 and 5
 # - return the value of `num` if `num` isn't divisible by either 3 or 5
 
-# Replace this comment with your code.
+min = #choose
```

Comment:
For this question, you just needed to write a method to calculate the response, e.g. "fizz", "buzz", "fizzbuzz", "other". No need to loop.